.github/workflows/push-check.yml

```diff
@@ -0,0 +1,14 @@
+name: push-check
+on: [push]
+jobs:
+  push-checks:
+    runs-on: ubuntu-latest
```

Comment:
meglio versionarlo :)

Reply:
in che senso?

Reply:
mettera la versione di ubuntu esplicita e non latest

Reply:
ah ok